frontend/src/profile/progress/ProgressTab.tsx

```diff
@@ -78,6 +83,63 @@ const ProgressTab: React.FC<ProgressTabProps> = ({ user, isCurrentUser }) => {
                 }
             }
         });
+
+        const suggestedTasks: Category = {
+            name: RequirementCategory.SuggestedTasks,
+            requirements: [],
+            totalComplete: 0,
+            totalRequirements: 0,
+        };
+
+        const desiredTaskCount = 3;
+        const randomTaskCount = 1;
+
+        const recentRequirements = Object.values(user.progress)
+            .toSorted(
+                (a, b) =>
+                    parseISO(b.updatedAt).getTime() - parseISO(a.updatedAt).getTime(),
```

Comment:
toSorted works via a comparing function though, right? And I'm sorting based not on the array element (which is not sortable being an object), but a field of it. I'm not sure here how to avoid converting to a number (unless javascript has a <=> operator that I'm missing).

Reply:
You can do `b.updatedAt.localeCompare(a.updatedAt)`

Reply:
Perfect! Thank you!